### DISCUSSION TITLE
Nested task lists

### DISCUSSION THREAD

**Discussion by xypnox** [vriteio/vrite#13] [Ideas]
Nested task lists in editor

Ex: 

- [ ] Hello there
  - [x] Nested task
  - [ ] And some more

**Comment by areknawo** [comment]
These will soon be available. I have to figure out some parsing issues.

**Comment by areknawo** [comment]
This is available from v2023.8.11 with new Input/Output Transformers, soon to be implemented in Astro integration and publishing extensions.
